content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -83,9 +83,8 @@ In the `.yaml` file for the Kubernetes object you want to create, you'll need to
 
 The precise format of the object `spec` is different for every Kubernetes object, and contains nested fields specific to that object. The [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/) can help you find the spec format for all of the objects you can create using Kubernetes.
 
-For example, the reference for Pod details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
-for a Pod in the API, and the reference for Deployment details the [`spec` field](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) for Deployments.
-In those API reference pages you'll see mention of PodSpec and DeploymentSpec. These names are implementation details of the Golang code that Kubernetes uses to implement its API.
+For example, see [this `spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) reference for Pod details, for a Pod in the API, and [this `spec` field](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) reference for Deployment details, for Deployments.
```

Comment:
👋 @alghe-global . I think the text could be clearer. Is this correct:

For example, see the details of the [Pod `spec`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) and the definition of the [Deployment `spec`](/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec) in the Kubernetes API reference. Kubernetes uses a resource's `spec` to implement its API.